semantic/src/site/globals/site.overrides

```diff
@@ -1728,60 +1726,6 @@ a:hover {
   }
 }
 
-/*--------------------------------
-      Data views
----------------------------------*/
-
-#data-view {
-  .flex-column();
-  align-items: center;
-  width: @applicationListWidth;
-
-  .ui.header {
-    padding-top: 30px;
```

Comment:
Makes sense to remove this and put it in table over-rides, but can you leave the header here and add a comment to see "Table" overrides file?

Reply:
Moved ui.header back to site.overrides.